lib/listeners/in.js

```diff
@@ -63,3 +75,48 @@ exports.callback = function (route, message, response) {
     });
   }
 };
+
+function getPrettyAnnounce (gameToAnnounce) {
+  var wDecorator = '';
```

Comment:
Can you switch `var` to `let` (if the value changes later on) or `const` (if it doesn't)?